ganga/GangaCore/Lib/Condor/Condor.py

```diff
@@ -718,12 +718,12 @@ def getStateTime(self, status):
             splitLine = l.split()
             if checkstr == splitLine[0]:
                 year = datetime.datetime.now().year
-                if datetime.datetime.strptime(str(datetime.datetime.now().year)+'/'+splitLine[2]+' '+splitLine[3], "%Y/%m/%d %H:%M:%S") > datetime.datetime.now():
+                if datetime.datetime.strptime(splitLine[2]+' '+splitLine[3], "%Y-%m-%d %H:%M:%S") > datetime.datetime.now():
```

Comment:
This change should be made such that the old format (without the year as far as I can see) is accepted as well.

Reply:
Right, I was not aware that we were still using an old version. I have written a small parser now that checks the date format the first time the date is parsed and stores it in _condorDateFormat. It counts the number of date elements and checks the separation character
Whenever the date from the log file is requested, the date reformatted according to _condorDateFormat.

Reply:
HTCondor is not used just at CERN so we should guard against this type of changes.